app/models/profile.rb

```diff
@@ -12,7 +12,8 @@ def ensure_image_from_github
 
   private def fetch_profile_image_from_github
     client = Octokit::Client.new
-    url = client.user(user.authentication_provider_github.uid.to_i).avatar_url
+    uid = user.authentication_provider_github&.uid || raise
+    url = client.user(uid.to_i).avatar_url
```

Comment:
https://github.com/kaigionrails/conference-app/blob/2fa62c3a3e5b1527e936657a9f2467d12eede260/sig/rbs_rails/app/models/user.rbs#L210
How about fix this line to `AuthenticationProviderGithub` ?

Reply:
I thought it's not good to rewrite the RBS files generated by rbs_rails because they will be re-generated in the future, and the manual changes will be lost.

But we can declare custom definitions on the `sig/handwritten/` directory. So it's much better than modifying the Ruby code. I'll withdraw this change and try another way.

Thanks,

Reply:
Fixed.